.circleci/config.yml

```diff
@@ -53,13 +49,11 @@ workflows:
   "@oclif/plugin-not-found":
     jobs:
       - node-latest
-      - node-12
-      - node-10
+      - node-14
       - release:
           context: org-global
           filters:
-            branches: {only: master}
+            branches: {only: main}
```

Comment:
I'm guessing we aren't merging any of these PRs until we have an `@oclif/core@v1`?